### PR TITLE
T/135: Implement a creator with sticky toolbar

### LINKED ISSUE #135: Implement a creator with sticky toolbar
The idea is to implement a creator which is a hybrid between CKEditor 4 ["inline" and "classic"](http://docs.ckeditor.com/#!/guide/dev_ckeditor_js_load) implementations:
* Inline content (`<div contenteditable="true"...`).
 * No content scrolling.
* "Classic" toolbar.
 * Always visible (regardless of focus).
 * 100% content width.
* Toolbar sticks to the edge of the viewport when content goes off screen, i.e. when a page is scrolled.

image

### PR DESCRIPTION
**Pull request by oleq**
Closes #135.

### CODE REVIEW

**Comment by Reinmar** [comment]
I have this:

```
11:34:55] Compiling 'ckeditor.css' from 2 entry points...
[11:34:55] Plumber found unhandled error:
 Error: no mixin named ck-editor

Backtrace:
	../ckeditor5-ui-default/theme/components/stickytoolbar.scss:4
```

**Comment by oleq** [comment]
That's strange. WFM
```
t/135 dfe55c2) oleq@MBP ~/CK/ckeditor5> gulp build:clean:all
[11:58:05] Using gulpfile ~/CK/ckeditor5/gulpfile.js
[11:58:05] Starting 'build:clean:all'...
[11:58:05] Deleted file '/Users/oleq/CK/ckeditor5/build'.
[11:58:05] Finished 'build:clean:all' after 217 ms
(t/135 dfe55c2) oleq@MBP ~/CK/ckeditor5> gulp build:themes
[11:58:09] Using gulpfile ~/CK/ckeditor5/gulpfile.js
[11:58:09] Starting 'build:themes'...
[11:58:09] Starting 'build:clean:themes'...
[11:58:09] Finished 'build:clean:themes' after 7.82 ms
[11:58:09] Starting 'build:icons'...
[11:58:10] Output for amd is '/Users/oleq/CK/ckeditor5/build/amd/theme/iconmanagermodel.js'.
[11:58:10] Finished 'build:icons' after 840 ms
[11:58:10] Starting 'build:sass'...
[11:58:10] Found theme entry point '/Users/oleq/CK/ckeditor5-theme-lark/theme/theme.scss'.
[11:58:10] Found theme entry point '/Users/oleq/CK/ckeditor5-ui-default/theme/theme.scss'.
[11:58:10] Compiling 'ckeditor.css' from 2 entry points...
[11:58:10] Output for amd is '/Users/oleq/CK/ckeditor5/build/amd/theme/ckeditor.css'.
[11:58:10] Finished 'build:sass' after 64 ms
[11:58:10] Finished 'build:themes' after 917 ms
(t/135 dfe55c2) oleq@MBP ~/CK/ckeditor5>
```

**Comment by Reinmar** [comment]
OK, I think that all branches are ready to merge them. Great job!

**Comment by Reinmar** [comment]
I was very close to merge this PR, but I found that few classes in https://github.com/ckeditor/ckeditor5-ui/pull/24 misses docs.

File: tests/_utils/ui/editableui/framed/framededitableuiiframeview.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	 */
+	constructor( model, locale ) {
+		super( model, locale );
+
+		const bind = this.attributeBinder;
+
+		this.template.attributes.class.push( 'ck-editor__editable-iframe' );
+		this.template.attributes.style = [
+			'width:',
+			bind.to( 'width', ( w ) => `${w}px` ),
```
Comment (Reinmar) [inline tests/_utils/ui/editableui/framed/framededitableuiiframeview.js]: This is bad... We need to find a better solution, which would be an object format for styles. We can't rewrite all styles every time. I'll create a followup for this.

Comment (Reinmar) [inline tests/_utils/ui/editableui/framed/framededitableuiiframeview.js]: Beside, it doesn't do anything with the editor – it only resizes the iframe while the container stays at 100%.

Comment (Reinmar) [inline tests/_utils/ui/editableui/framed/framededitableuiiframeview.js]: As for the followup: https://github.com/ckeditor/ckeditor5-ui/issues/27

**Pull request merged**